### PR TITLE
Add comments to structurator

### PR DESCRIPTION
**Pull request by Timotshak**

### CODE REVIEW

File: sampo/structurator/__init__.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
 from sampo.structurator.base import graph_restructuring, STAGE_SEP
 from sampo.structurator.graph_insertion import graph_in_graph_insertion
 from sampo.structurator.light_modifications import work_graph_ids_simplification
-from sampo.structurator.base import graph_restructuring
```
Comment (StannisMod) [inline sampo/structurator/__init__.py:4]: Why?

**Review by StannisMod — CHANGES_REQUESTED** [review]
Please answer the question

File: sampo/structurator/__init__.py (hunk 1 above)
Comment (Timotshak) [reply sampo/structurator/__init__.py:4]: because this is a repeat of the import that was already done in the first line

**Review by StannisMod — APPROVED** [review]
Approved

**Pull request merged**